### ISSUE TITLE
How to modify the algorithms loss function?

### ISSUE THREAD

**Issue by adebiasio-estilos** [massquantity/LibRecommender#256]
Dear All,
First of all thank you very much for this amazing library! The work is really remarkable and well done!

As per title I write because I would like to understand how to modify the loss function of some simple algorithms through this library.

- Here is provided an example to modify the loss function of BPR (according to importance sampling weight): https://github.com/massquantity/LibRecommender/issues/203
- Is there a similar way to modify the binary cross entropy loss function of SVD to introduce some weighting for example?

I found that here https://github.com/massquantity/LibRecommender/blob/master/libreco/tfops/loss.py are declared some loss functions.
Suppose I would like to use a custom loss that weight the binary cross entropy for an item's weight. What should I do to integrate it into the library?

**Comment by massquantity** [comment]
You can replace the [tf.nn.sigmoid_cross_entropy_with_logits](https://github.com/massquantity/LibRecommender/blob/master/libreco/tfops/loss.py#L15) with [tf.nn.weighted_cross_entropy_with_logits](https://www.tensorflow.org/versions/r1.15/api_docs/python/tf/nn/weighted_cross_entropy_with_logits).

**Comment by adebiasio-estilos** [comment]
great, thank you very much!
